resources/styleheets/main.css

```diff
@@ -0,0 +1,276 @@
+  border-top: 1px solid #ccc;
+}
+
+.text {
+  text-align: left;
+}
+.text p {
+  font-size: 18px;
+}
+
+@media only screen and (max-width: 800px) {
+  .hidden {
+    display: none;
+  }
+}
+@media only screen and (min-width: 800px) {
+  .text .read-more-btn.hidden_ {
+    display: none;
+  }
+}
+.read-less-btn {
+  display: none;
+}
+
+.image img {
+  max-width: 100%;
+  height: auto;
+  border-radius: 25px;
+}
+
+.actions {
+  display: flex;
+  align-items: center;
+  font-size: 20px;
+}
+.actions p {
+  margin-right: 15px;
+  cursor: point;
+  transition: background-color 0.3s, color 0.3s;
+}
+.actions p:hover {
+  color: #06138a;
+}
+
+nav {
+  z-index: 1;
+  margin-top: 60px;
+  position: fixed;
+  width: 13%;
+  height: 100%;
+  min-width: 120px;
+  background-color: #262525;
+  color: #ccc;
+  box-shadow: 0px 2px 4px rgba(0, 0, 0, 0.5);
+  overflow-y: auto;
+  transition: transform 0.3s ease-in-out;
+  font-size: 20px;
+}
+nav .line {
+  display: flex;
+  align-items: center;
+  padding: 10px;
+  border-top: 1px solid #ccc;
+}
+nav .icons {
+  color: #ccc;
+  display: flex;
+  flex-direction: column;
+  padding: 0;
+  margin: 0;
+  list-style: none;
+}
+nav .icons a {
+  color: #ccc;
+  text-decoration: none;
+}
+nav .icons a .category {
+  position: relative;
+  display: flex;
+  align-items: center;
+  padding: 8px;
+  cursor: pointer;
+  transition: background-color 0.3s, color 0.3s;
+  color: #ccc;
+}
+nav .icons a .category:hover {
+  background-color: white;
+  color: black;
+}
+nav .icons a span {
+  margin-right: 15px;
+}
+nav .icons .categories {
+  position: relative;
+  display: flex;
+  align-items: center;
+  padding: 8px;
+  cursor: pointer;
+  transition: background-color 0.3s, color 0.3s;
+  color: #ccc;
+}
+nav .icons .categories:hover {
+  background-color: white;
+  color: black;
+}
+
+@media only screen and (max-width: 1250px) {
+  nav {
+    font-size: 13px;
+  }
+}
+@media only screen and (max-width: 800px) {
+  nav {
+    width: 100%;
+    height: 35%;
+    position: fixed;
+    top: 0;
+  }
+  .category p {
+    margin: 0;
+    font-size: 20px;
+  }
+  .line {
+    display: flex;
+    align-items: center;
+    padding: 6px;
+    border-top: 1px solid #ccc;
+  }
+  .line {
+    display: flex;
+    align-items: center;
+    padding: 6px;
+    border-top: 1px solid #ccc;
+  }
+}
+.hide-nav {
+  display: none;
+}
+
+.subcategories {
+  display: none;
+  position: absolute;
+  width: 200px;
+  top: 0;
+  left: 100%;
+  list-style: none;
+  padding: 0;
+  margin: 0;
+  background-color: #262525;
+  flex-direction: column;
+  z-index: 1;
+}
+
+.subcategories .subcategory {
+  padding: 10px;
+  cursor: pointer;
+  font-size: 16px;
+  transition: background-color 0.3s, color 0.3s;
+}
+
+.subcategories .subcategory:hover {
+  background-color: white;
+  color: green;
+}
+
+.subcategories a {
+  text-decoration: none;
+  color: inherit;
+}
+
+.subcategories span {
+  margin-right: 10px;
+}/*# sourceMappingURL=main.css.map */
```

Comment:
new line at the end of the file, and remember leave a new line between each block